storage/README.md

```diff
@@ -0,0 +1,4 @@
+collection of different storage interfaces that are shared by two or more components.
```

Comment:
Should we rewrite to:

A collection of different storage interfaces that are shared by two or more components